docs/configuration.asciidoc

```diff
@@ -134,7 +134,7 @@ Below is a sample `Web.config` configuration file for a ASP.NET application.
 This is used to keep all the errors and transactions of your service together
 and is the primary filter in the Elastic APM user interface.
 
-NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+$. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces. Characters in service name which don't match regular expression will be replaced by `__` symbol.
+NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9_-]+$`. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces. Characters in service name which don't match regular expression will be replaced by `__` symbol.
```

Comment:
Is this change ok? I removed the space between the `9` and the `_` because the formatting goes wild if there's a space. I also put this in a code snippet for easier readability.

Reply:
Could you clarify why space needs to be removed from `^[a-zA-Z0-9 _-]+$`? I see that other agents have it and in the paragraph above the following sentence says that spaces are allowed.

Reply:
My mistake -- thanks for catching this Sergey. You're right, removing the space from the regular expression doesn't make sense as then spaces wont match.
```suggestion
NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]+$`. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces. Characters in service name which don't match regular expression will be replaced by `_` symbol.
```